### PR TITLE
Standardize openshift roles to OASIS

### LINKED ISSUE #1657: Standardize OpenShift role to OASIS
**Is your feature request related to a problem? Please describe.**

This would require us to standardize the linchpin OpenShift role to OASIS.
so that eventually it can be imported to ansible-galaxy.

### PR DESCRIPTION
**Pull request by samvarankashyap**
$Subject

fixes #1657

### CODE REVIEW

File: linchpin/provision/roles/openshift/molecule/docker/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: openshift
+    image: centos:7
+    ## Remove 'image' above and uncomment the following to run systemd in Docker
```
Comment (14rcole) [inline linchpin/provision/roles/openshift/molecule/docker/molecule.yml]: You can remove this

File: linchpin/provision/roles/openshift/tasks/provision_openshift.yml [hunk 2]
```diff
@@ -82,23 +81,28 @@
 
 - name: fail when pod_output is failure
   fail:
-    msg: "Multiple pods in pending/failure state please adjust the timeout value and retry"
-  when: state == "present" and pod_output == 'Failure' 
+    msg: "Multiple pods in pending/failure \
```
Comment (14rcole) [inline linchpin/provision/roles/openshift/tasks/provision_openshift.yml]: Are you sure this works the way you expect it to?  Generally in yaml the way to split a string across multiple lines is this:
```
msg: >
  Multiple pods in pending/failure state, please adjust
  the timeout value and retry
```

Comment (samvarankashyap) [reply linchpin/provision/roles/openshift/tasks/provision_openshift.yml]: @14rcole I tested it with a simple debug msg task. I went fine. 
I can change it to the above format if its the general standard.

Comment (14rcole) [reply linchpin/provision/roles/openshift/tasks/provision_openshift.yml]: I did the above for the other modules, but we can standardize on whichever one you prefer

Comment (samvarankashyap) [reply linchpin/provision/roles/openshift/tasks/provision_openshift.yml]: @14rcole When I use the above syntax ansible lint was failing.

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**